suitcase/pizza-box/handlers.py

```diff
@@ -0,0 +1,116 @@
+import uuid
+import humanize
+import pandas as pd
+import os
+from databroker.tests.utils import temp_config
+from databroker import Broker
+from databroker.assets.handlers_base import HandlerBase
```

Comment:
You can just skip `HandlerBase` and inherit from nothing -- that is, remove this import and change:

```py
class PizzaBoxANHandler(HandlerBase):
```

to

```py
class PizzaBoxANHandler:
```

`HandlerBase` seemed like it might be a good idea years ago, but we have since discovered it's not necessary.